sw/device/rom_exts/development_keys_and_signatures/readme.md

```diff
@@ -0,0 +1,82 @@
+# Introduction
+
+The main idea of the development key pair and other signing artefacts is to
+make cross-development of on device and host side components for ROM_EXT
+signing easier.
+
+It is also useful when starting new development, as it is easier to make sure
+that signing process is correct by matching the produced  hashes and signatures
+against the reference files provided.
+
+Typical use-case would be - signing ROM_EXT image via host side tooling, and
+then using the same key pair on device to validate the signature.
+
+Files breakdown:
+- `dev_key_private.pem`    : private key in PEM representation.
+- `dev_key_private.der`    : private key in DER encoded binary representation.
+- `dev_key_public.pem`     : public key in PEM representation.
+- `dev_key_public.der`     : public key in DER encoded binary representation.
+- `hello_world.bin`        : reference file containing "Hello World!".
+- `hello_world_digest.bin` : SHA-256 digest of the reference file.
```

Comment:
nit: Just for separation of files, it would be nice if you could move this to a testing directory, you could have the following directory structure: 

*    `rom_exts/keys`: This is where the keys live.
*    `rom_exts/keys/testing`: This is where you have all the reference binaries for testing, e.g. `hello_world*`.

Reply:
Done.